samples/cxx/flamespeed2/Makefile

```diff
@@ -0,0 +1,20 @@
+include /usr/local/include/cantera/Cantera.mak
```

Comment:
The Makefile should be generated automatically by SCons, by adding this example to the list in `samples/cxx/SConscript`